docs/src/PolyhedralGeometry/Polyhedra/pg_polyhedra_polymake.md

```diff
@@ -0,0 +1,55 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["pg_polyhedra_polymake.md"]
+```
+
+
+# `Polyhedron` and `polymake`'s `Polytope`
+
+Many polyhedral computations are done through `polymake`.
+This is visible in the structure `Polyhedron` via a pointer `pm_polytope` to the corresponding `polymake` object.
+This allows to apply all functionality of `polymake` by calling a suitable `Polymake.jl` function on this pointer.
+
+To allow both `Oscar`'s and `polymake`'s functionality to be applicable to a polyhedron object, it can be converted back and forth:
```

Comment:
I think that's a rather unfortunate way to state things, given that Oscar is built on polymake, via Polymake.jl, while this makes it sound as if they were two separate systems. 

Also, I hope these are not really doing conversions, but rather are about wrapping and unwrapping objects  Polymake.jl.

So, perhaps like this:
```suggestion
Sometimes it can be necessary to directly invoke some `polymake` functions on an Oscar `Polyhedron` object (e.g. because some functionality has not yet been made available via Oscar's interface). In that case, the following two functions allow extracting the underlying `Polymake.jl` object from a `Polyhedron`, respectively wrapping a `Polymalke.jl` object representing a polyhedron into a high-level `Polyhedron` object.
```